### PR TITLE
Add contract fees

### PR DESCRIPTION
**Pull request by jacogr**
Pull all contract fees, part of https://github.com/polkadot-js/apps/issues/983

### CODE REVIEW

File: packages/api-derive/src/balances/fees.ts [hunk 1]
```diff
@@ -16,12 +16,8 @@ export function fees (api: ApiInterface$Rx) {
     return (combineLatest([
       api.query.balances.creationFee(),
       api.query.balances.existentialDeposit(),
-      api.query.fees
-        ? api.query.fees.transactionBaseFee()
-        : api.query.balances.transactionBaseFee(),
-      api.query.fees
-        ? api.query.fees.transactionByteFee()
-        : api.query.balances.transactionByteFee(),
+      api.query.balances.transactionBaseFee(),
```
Comment (jacogr) [inline packages/api-derive/src/balances/fees.ts]: Fees modules has been removed for a long time now, safe to remove.

Comment (Stefie) [reply packages/api-derive/src/balances/fees.ts]: Is that how you'd solve it for all deprecated methods?

Comment (jacogr) [reply packages/api-derive/src/balances/fees.ts]: Depends on the actual case. So `transactionBaseFee` used to be in balances, then got moved to fees (and some networks supported it, some others not) and then got moved back from fees to balances.

In this case is was _exactly_ the same, just the module changed. (And quite easy to detect)

Same has gone in the past with the split into the `indices` modules (the detection there is already removed). These are easy cases - where the functionality ceases to exist, it is better just having a new file where things are done differently. (e.g. old intentions vs new controllers in staking)

Generally when this happens, it is viewed as a short-term override. As soon as the official Polkadot & Substrate test networks have been updated, it _should_ be removed. (Sadly these have not been logged each and every time - generally we do log these "support multiples" as issues, but I believe this specific one slipped through)

Comment (Stefie) [reply packages/api-derive/src/balances/fees.ts]: Ok, so I'll add `support multiples`  to mine. Maybe with a date or commit hash of when it was changed in substrate master?

**Pull request merged**